### ISSUE TITLE
Mock API responses for testing

### ISSUE THREAD

**Issue by nightsh** [tenders-exposed/elvis-ember#258]
We need to mock a lot of things in order to be able to test the app properly.

- accounts:
  - [ ] creation
  - [ ] recovery
  - [ ] confirmation
  - [ ] update
- networks
  - [ ] query
  - [ ] show
  - [ ] update
- cpvs
  - [ ] sample payload for creating a tree
- years and countries
  - [ ] sample payload for rendering components

There might be some more, these are just what I've encountered so far. Related to the recent tests refactoring and removal of old, deprecated tests.

**Comment by victor-nitu** [comment]
Blocked by backend migration, as payload structure will change drastically.

**Comment by victor-nitu** [comment]
Closing for now, as WONTFIX
